src/main/java/frc/robot/Robot.java

```diff
@@ -34,20 +36,28 @@ public void robotInit() {
     // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
     // autonomous chooser on the dashboard.
     m_robotContainer = new RobotContainer();
+    m_autonomousCommand = m_robotContainer.getAutonomousCommand();
   }
 
   /**
    * This function is called every robot packet, no matter the mode. Use this for items like
    * diagnostics that you want ran during disabled, autonomous, teleoperated and test.
    *
    * <p>This runs after the mode specific periodic functions, but before
-   * LiveWindow and SmartDashboard integrated updating.
+   * LiveWindow and SmartDashboard integrated updatin
+   *g.
    */
   @Override
   public void robotPeriodic() {
     SmartDashboard.putNumber("Left Velocity", -1 * m_robotContainer.mDriveSubsystem.leftVelocity());    
     SmartDashboard.putNumber("Right Velocity", -1 * m_robotContainer.mDriveSubsystem.rightVelocity());
     m_robotContainer.mDriveSubsystem.updatePID();
+    double x = m_robotContainer.tx.getDouble(0.0);
+    double y = m_robotContainer.ty.getDouble(0.0);
+    double area = m_robotContainer.ta.getDouble(0.0);
```

Comment:
Since tx, ty, ta, and mDriveSubsystem are static variables declared in the RobotContainer class, they should be accessed using the class name rather than the instance name. 
i.e. RobotContainer.tx.getDouble(0.0) instead of m_robotContainer.tx.getDouble(0.0)